fs/span-manager/span_manager.go

```diff
@@ -38,6 +38,30 @@ var (
 	ErrExceedMaxSpan       = errors.New("span id larger than max span id")
 )
 
+type MultiReaderCloser struct {
+	c []io.Closer
+	io.Reader
```

Comment:
Would this need to be pointer embedded?

Reply:
I don't think so since it's an interface, which also handles pointers.

Reply:
It was need for `io.SectionReader` since it returns a concrete pointer type.

Reply:
Ah, I see. Yeah was reader `io.SectionReader` and trying to understand the difference. Thanks for answering.